test/utils.cpp

```diff
@@ -325,6 +377,13 @@ TEST_CASE("absolute_url()", "[utils]")
 			"bla2.html") == "http://test:test@foobar:33/bla2.html");
 }
 
+TEST_CASE("quote_for_stfl()", "[utils]")
+{
+	REQUIRE(utils::quote_for_stfl("<<><><><") == "<><>><>><>><>");
```

Comment:
Depending on what you'll write in the test description, it might be worthwhile to put these checks into separate sections and give them more detailed descriptions.

Reply:
Thank you, but I don't think that will be necessary. Let me know what you think.

Reply:
I think test description says enough, no need for sections. Thanks!